Gemfile

```diff
@@ -0,0 +1,8 @@
+source "https://rubygems.org"
+
```

Comment:
what if we just `gempsec` here?

Reply:
Since `rubygems-update` does not have any runtime dependencies, I don't think it's necesary?